code/game/objects/structures/tables_racks.dm

```diff
@@ -720,6 +720,9 @@
 		if(computer)
 			computer.table = src
 			break
+
+	RemoveElement(/datum/element/climbable)
```

Comment:
Why is this being removed? Is it because it conflicts with buckling mechanics?

Reply:
Yes, that is exactly why. It's a click+drag to buckle yourself and a click+drag to climb.